<launch>
  <!-- <arg name="serial_port" default="/dev/ttyUSB0"/> -->

  <node pkg="nortek_dvl_driver" exec="nortek_dvl_driver_node" 
        name="nortek_dvl_driver_node" namespace="sensors">
    <!-- <param name="serial_port" value="$(var serial_port)"/> -->
  </node>
</launch>